Assets/Scripts/UI/MainMenu/MainMenuController.cs

```diff
@@ -367,6 +367,8 @@ public void FetchLobbyInfo()
 
     public void FetchQueueLobbyInfo()
     {
+        if (!SteamManager.IsSteamActive)
```

Comment:
Perhaps this should reside within the method in SteamManager?

Reply:
Sure